### ISSUE TITLE
Does the final model needs Normalization?

### ISSUE THREAD

**Issue by MasoudKaviani** [OpenGVLab/InternVideo#42]
I read the code and saw that the final embedding did not normalize.
Does it a good idea to normalize the final embedding vector?

**Comment by liyz15** [comment]
Thank you for trying out InternVideo!

We normalize the embedding in the loss when training. Please refer to the [demo](https://github.com/OpenGVLab/InternVideo/blob/0df66a6ba3613469b4eb3602b4f556422b432c12/Pretrain/Multi-Modalities-Pretraining/demo.py#L16) for use case.
